test/shared/exports.test.js

```diff
@@ -12,6 +12,5 @@ describe('preact', () => {
 		expect(cloneElement).to.be.a('function');
 		expect(createContext).to.be.a('function');
 		expect(options).to.exist.and.be.an('object');
-		// expect(rerender).to.be.a('function');
```

Comment:
I think it is better to describe  in the release note that `rerender()` was removed.

Reply:
Good catch, added it to the release notes under breaking changes :+1: